Code/autopkglib/pyads/pyads.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+"""
+Copyright © 2015, Robin David - MIT-Licensed
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+The Software is provided "as is", without warranty of any kind, express or implied, including but not limited
+to the warranties of merchantability, fitness for a particular purpose and noninfringement. In no event shall
+the authors or copyright holders X be liable for any claim, damages or other liability, whether in an action
+of contract, tort or otherwise, arising from, out of or in connection with the software or the use or other
+dealings in the Software.
+Except as contained in this notice, the name of the Robin David shall not be used in advertising or otherwise
+to promote the sale, use or other dealings in this Software without prior written authorization from the Robin David.
+"""
+import os
+import sys
+from ctypes import *
```

Comment:
This doesn't belong as a file in the project itself, but rather a pip dependency.

Reply:
Bad idea...
This module does a complete different thing (it is for PLC API)!
Unfortunateley, they are both named the same.
There is no module for pyads for alternate data streams, i am aware of.

Reply:
oh you are right, this is a different pip library. So where is this one you are using from?